packages/manager/core/generator/app/templates/src/components/Breadcrumb.tsx

```diff
@@ -21,43 +28,39 @@ type Match = {
   params: Params<string>;
 };
 
-export default function BreadcrumbComponent(): JSX.Element {
+function Breadcrumb(): JSX.Element {
   const matches = useMatches();
-  const [crumbs, setCrumbs] = useState<Record<string, string>[]>();
+  const [crumbs, setCrumbs] = useState<BreadcrumbItem[]>([]);
 
   useEffect(() => {
     const items = matches.map(async (match) => {
       const { handle, data, params }: Match = match;
       const breadcrumb = await handle?.breadcrumb;
-      return {
-        pathname: match.pathname,
-        crumb: await breadcrumb?.({ data, params }),
+      const crumb = {
+        label: await breadcrumb?.({ data, params }),
+        href: match.pathname,
       };
+      return crumb;
     });
 
     Promise.all(items)
       .then((breadcrumbs) => {
-        setCrumbs(breadcrumbs.filter(({ crumb }) => !!crumb));
+        const validCrumbs = breadcrumbs.filter(
+          ({ label, href }) => !!label && !!href,
+        );
+        setCrumbs(validCrumbs);
       })
-      .catch((err) => console.error(err));
+      .catch((error) => {
+        console.error('Error fetching breadcrumbs:', error);
+      });
   }, [matches]);
 
   // no need to display a breadcrumb for a single item
-  if (!crumbs || crumbs.length <= 1) {
-    return <></>;
+  if (crumbs?.length <= 1) {
+    return null;
   }
 
```

Comment:
It's repetitive with the ternary in the second return : you should remove this code

Reply:
Is it possible to return boolean false ?